velox/substrait/tests/VeloxSubstraitRoundTripTest.cpp

```diff
@@ -441,6 +443,21 @@ TEST_F(VeloxSubstraitRoundTripTest, arrayLiteral) {
       "array[], array[array[1,2,3], array[4,5]]");
 }
 
+TEST_F(VeloxSubstraitRoundTripTest, date1) {
```

Comment:
naming: date1 -> data (naming variables and methods using names like a1, a2, a3 is not considered best practice)

Reply:
@mbasmanova I didn't run the cleanup task yet. I will definitely put a better naming.